notebooks/official/sdk/sdk2_bigframes_sklearn.ipynb

```diff
@@ -0,0 +1,726 @@
+{
```

Comment:
<p>What is the convention for custom jobs/staging bucket artifacts for sample notebooks? Are they usually cleaned up?</p>

Reply:
<p>It's typically recommended to use job.delete to remove the training job. You can ignore if it doesn't apply to your case here.</p>

Reply:
<p>Got it, I would say it's not relevant here since the custom job is an implementation detail. Meaning that the job object isn't exposed to the user so it's not easy to delete.</p>